tests/generic_config_updater/test_incremental_qos.py

```diff
@@ -8,14 +8,27 @@
 from tests.common.helpers.dut_utils import verify_orchagent_running_or_assert
 from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_res_success, expect_op_failure
 from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 't1'),
     pytest.mark.asic('mellanox')
```

Comment:
>mellanox [](http://example.com/codeflow?start=22&length=8)

Is it safe to run this test on mellanox switch but not 2700?

Reply:
These formula are only applicable to 2700

Reply:
@wangxin Is there a marker we can add for 2700?

Reply:
Do you only want this test to run on 2700? If so, you can add a pytest_requires condition in your test code entrance to only allow 2700. But why?

Reply:
Yes, the Mellanox SPC1 buffer calculations only apply to 2700 devices. @neethajohn may confirm/correct me

And thanks, I added `pytest_require` to check for 2700 in hwsku